configs/default/BEFH-BETAFPVF722.config

```diff
@@ -0,0 +1,115 @@
+# Betaflight / STM32F7X2 (S7X2) 4.1.0 Oct 16 2019 / 11:58:45 (c37a7c91a) MSP API: 1.42
+
+board_name BETAFPVF722
+manufacturer_id BEFH
+
+# resources
+resource BEEPER 1 C14
+resource MOTOR 1 B00
+resource MOTOR 2 B01
+resource MOTOR 3 A01
+resource MOTOR 4 A00
+resource MOTOR 5 B06
+resource MOTOR 6 B07
+
+resource LED_STRIP 1 A08
+resource SERIAL_TX 1 A09
+resource SERIAL_TX 2 A02
+resource SERIAL_TX 3 B10
+resource SERIAL_TX 4 C10
+resource SERIAL_TX 5 C12
+resource SERIAL_TX 6 C06
+resource SERIAL_RX 1 A10
+resource SERIAL_RX 2 A03
+resource SERIAL_RX 3 B11
+resource SERIAL_RX 4 C11
+resource SERIAL_RX 5 D02
+resource SERIAL_RX 6 C07
+resource I2C_SCL 1 B08
+resource I2C_SDA 1 B09
+resource LED 1 C15
+resource SPI_SCK 1 A05
+resource SPI_SCK 2 B13
+resource SPI_SCK 3 B03
+resource SPI_MISO 1 A06
+resource SPI_MISO 2 B14
+resource SPI_MISO 3 B04
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 2 B15
+resource SPI_MOSI 3 B05
+resource CAMERA_CONTROL 1 C08
+resource ADC_BATT 1 C00
+resource ADC_RSSI 1 C02
+resource ADC_CURR 1 C01
+resource FLASH_CS 1 B12
+resource BARO_CS 1 C13
+resource OSD_CS 1 A15
+resource GYRO_EXTI 1 C04
+resource GYRO_CS 1 A04
+
+# timer
+
+timer B00 AF2
+# pin B00: TIM3 CH3 (AF2)
+timer B01 AF2
+# pin B01: TIM3 CH4 (AF2)
+timer A00 AF1
+# pin A00: TIM2 CH1 (AF1)
+timer A01 AF1
+# pin A01: TIM2 CH2 (AF1)
+timer B06 AF2
+# pin B06: TIM4 CH1 (AF2)
+timer B07 AF2
+# pin B07: TIM4 CH2 (AF2)
+timer A08 AF1
+# pin A08: TIM1 CH1 (AF1)
+timer C08 AF3
+# pin C08: TIM8 CH3 (AF3)
+
+# dma
+dma ADC 1 0
+# ADC 1: DMA2 Stream 0 Channel 0
+dma pin B00 0
+# pin B00: DMA1 Stream 7 Channel 5
+dma pin B01 0
+# pin B01: DMA1 Stream 2 Channel 5
+dma pin A00 0
+# pin A00: DMA1 Stream 5 Channel 3
+dma pin A01 0
+# pin A01: DMA1 Stream 6 Channel 5
+dma pin B06 0
+# pin B06: DMA1 Stream 0 Channel 2
+dma pin B07 0
+# pin B07: DMA1 Stream 3 Channel 2
+dma pin C08 0
+# pin C08: DMA2 Stream 4 Channel 7
+dma pin A08 0
+# pin A08: DMA2 Stream 3 Channel 6
+
+# serial
+serial 2 64 115200 57600 0 115200
+
+# feature
+feature -RX_PARALLEL_PWM
+feature RX_SERIAL
+feature OSD
+
+# master
+
+set baro_spi_device = 3
+set serialrx_provider = SBUS
+set blackbox_device = SPIFLASH
+set dshot_idle_value = 450
+set motor_pwm_protocol = DSHOT600
```

Comment:
Unless this is only going to be used for an all-in-one board with integrated ESCs, this default setting is a safety risk, as it will result in spinning motors on power up when used with ESCs that do not support Dshot.
Is this target going to be used only for an all-in-one board?

Reply:
Although the first one is an all-in-one board, I can't guarantee that it will only be used for an all-in-one board, so I will remove set motor_pwm_protocol = DSHOT600.

Reply:
Although the first one is an all-in-one board, I can't guarantee that it will only be used for an all-in-one board, so Dshot600 can be removed.